libs/net/dal/Models/UserFilter.cs

```diff
@@ -32,6 +34,8 @@ public UserFilter(Dictionary<string, StringValues> queryParams) : base(queryPara
         this.IsEnabled = filter.GetBoolNullValue(nameof(this.IsEnabled));
         this.Status = filter.GetEnumNullValue<Entities.UserStatus>(nameof(this.Status));
         this.IsSystemAccount = filter.GetBoolNullValue(nameof(this.IsSystemAccount));
+        this.Keyword = filter.GetStringValue(nameof(this.Keyword));
+        this.RoleName = filter.GetStringValue(nameof(this.RoleName));
```

Comment:
Shouldn't this be nullable?  `filter.GetStringNullValue(...)`?

Reply:
We don't have an extension method for `GetStringNullValue`, should I create one or do we not need one for strings?

Reply:
Hmmm looks like the `GetStringValue` returns a nullable string.  Not ideal, but normal before we had the `?` nullable syntax option.
We should probably update this, but it would require changes unrelated to this PR.  Let's leave it now as tech debt.